coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalHierarchyServiceImpl.java

```diff
@@ -349,6 +349,18 @@ public void synchronizeAllChildren(DevelopmentProposal hierarchyProposal) throws
         prepareHierarchySync(hierarchyProposal);
         synchronizeAll(hierarchyProposal);
     }
+    
+    public void synchronizeAllChildrenBudgets(DevelopmentProposal hierarchyProposal) throws ProposalHierarchyException {
+    	prepareHierarchySync(hierarchyProposal);
+        finalizeHierarchySync(hierarchyProposal.getProposalDocument());
+
+        for (DevelopmentProposal childProposal : getHierarchyChildren(hierarchyProposal.getProposalNumber())) {
+            List<BudgetPeriod> oldBudgetPeriods = getOldBudgetPeriods(getHierarchyBudget(hierarchyProposal));
+            synchronizeChildBudget(hierarchyProposal, childProposal, oldBudgetPeriods);
+            dataObjectService.save(childProposal);
```

Comment:
So returning a value here didn't do anything different. Do you still want me to do that? Across the hierarchy service?